src/DIRAC/FrameworkSystem/Service/SystemAdministratorHandler.py

```diff
@@ -13,9 +13,16 @@
 import getpass
 import importlib
 import shutil
+import platform
 import psutil
+import tempfile
 
 import six
+from packaging.version import Version, InvalidVersion
+try:
+  import subprocess32 as subprocess
+except ImportError:
+  import subprocess
 
 # TODO: This should be modernised to use subprocess(32)
```

Comment:
remove this comment.

Reply:
Why? This comment still applies as `commands` is used by `export_getHostInfo` and should be modernised at some point.

Reply:
a-ha, I missed that.